source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -41,6 +41,13 @@ Following enums were defined :
 - NIDCPOWER_VAL_CANCEL
```

Comment:
This is not your change, but looks like these two lines call out specific enum values. Can you replace these two lines with name of the enum `CalibrationCloseAction` instead?

Reply:
Done.